### PR TITLE
Change the default values of multiple tiflash configs

### PR DESCRIPTION
**Pull request by hehechen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update tiflash-configuration.md,.
1. Add description of tiflash-proxy apply-pool-size and store-pool-size configuration. These two configurations have existed since TiDB 4.0 version, this PR is just to supplement the description of the configuration. 
2. TiFlash has [changed default values of some configurations](https://github.com/pingcap/tics/pull/3414), so the doc need to be updated.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -176,6 +176,10 @@ delta_index_cache_size = 0
 [server]
     engine-addr = 外部访问 TiFlash coprocessor 服务的地址
 [raftstore]
+    ## 处理 Raft 数据落盘的线程池中线程的数量
+    apply-pool-size = 4
+    ## 表示处理 Raft 的线程池中线程的数量，即 Raftstore 线程池的大小。
+    store-pool-size = 4
     ## 控制处理 snapshot 的线程数，默认为 2。设为 0 则关闭多线程优化
     snap-handle-pool-size = 2
```
Comment (solotzg) [inline tiflash/tiflash-configuration.md:184]: Is it still 2 ?

Comment (hehechen) [reply tiflash/tiflash-configuration.md]: It is 4, I have changed it.

**Review by solotzg — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -130,17 +130,21 @@ delta_index_cache_size = 0
     advertise-status-addr = 外部访问 status-addr 的地址，不填则默认是 status-addr
 
 [logger]
-    level = log 级别（支持 trace、debug、information、warning、error）
+    ## log 级别（支持 trace、debug、information、warning、error），默认是 debug
+    level = debug
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:134]: 这里是否涉及默认值的更改？

Comment (hehechen) [reply tiflash/tiflash-configuration.md:134]: 不涉及，默认值本来就是debug

Comment (TomShawn) [reply tiflash/tiflash-configuration.md:134]: 但是之前这个配置模板里写的是【level = log】，这个不表示默认值吗？

Comment (hehechen) [reply tiflash/tiflash-configuration.md:134]: > 但是之前这个配置模板里写的是【level = log】，不表示默认值吗？

之前这个文件写的是`level = log 级别（支持 trace、debug、information、warning、error）`，没有写level的默认值是什么。

Comment (hehechen) [reply tiflash/tiflash-configuration.md:134]: level这里没有修改默认值。但是`count`，还有[status]下面的`metrics_port`，[raftstore]下面的`snap-handle-pool-size`修改了默认值。

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -176,8 +180,12 @@ delta_index_cache_size = 0
 [server]
     engine-addr = 外部访问 TiFlash coprocessor 服务的地址
 [raftstore]
-    ## 控制处理 snapshot 的线程数，默认为 2。设为 0 则关闭多线程优化
-    snap-handle-pool-size = 2
+    ## 处理 Raft 数据落盘的线程池中线程的数量
+    apply-pool-size = 4
+    ## 表示处理 Raft 的线程池中线程的数量，即 Raftstore 线程池的大小。
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:185]:
```suggestion
    ## 处理 Raft 的线程池中线程的数量，即 Raftstore 线程池的大小。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -176,6 +180,10 @@ delta_index_cache_size = 0
 [server]
     engine-addr = 外部访问 TiFlash coprocessor 服务的地址
 [raftstore]
+    ## 处理 Raft 数据落盘的线程池中线程的数量
+    apply-pool-size = 4
+    ## 处理 Raft 的线程池中线程的数量，即 Raftstore 线程池的大小。
+    store-pool-size = 4
     ## 控制处理 snapshot 的线程数，默认为 2。设为 0 则关闭多线程优化
     snap-handle-pool-size = 2
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:188]: @hehechen 
```suggestion
    snap-handle-pool-size = 4
```

File: tiflash/tiflash-configuration.md [hunk 5]
```diff
@@ -130,17 +130,21 @@ delta_index_cache_size = 0
     advertise-status-addr = 外部访问 status-addr 的地址，不填则默认是 status-addr
```
Comment (LiSong0214) [inline tiflash/tiflash-configuration.md:131]: [flash.proxy] addr 默认值由 127.0.0.1:20106 改成 127.0.0.1:20170，可以在手册中说明一下现有默认值是多少。

**Review by LiSong0214 — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 6]
```diff
@@ -120,27 +120,31 @@ delta_index_cache_size = 0
...
 [logger]
-    level = log 级别（支持 trace、debug、information、warning、error）
+    ## log 级别（支持 trace、debug、information、warning、error），默认是 debug
+    level = debug
     log = TiFlash log 路径
     errorlog = TiFlash error log 路径
-    size = 单个日志文件的大小
-    count = 最多保留日志文件个数
+    ## 单个日志文件的大小，默认是 100 MB
+    size = "100M"
```
Comment (qiancai) [inline tiflash/tiflash-configuration.md:138]: @hehechen  size = "100M" 这里应该是 100M 还是 100MB

File: tiflash/tiflash-configuration.md [hunk 7]
```diff
@@ -120,27 +120,31 @@ delta_index_cache_size = 0
...
 
 [logger]
-    level = log 级别（支持 trace、debug、information、warning、error）
+    ## log 级别（支持 trace、debug、information、warning、error），默认是 "debug"
+    level = debug
     log = TiFlash log 路径
     errorlog = TiFlash error log 路径
-    size = 单个日志文件的大小
-    count = 最多保留日志文件个数
+    ## 单个日志文件的大小，默认是 "100 M"
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:137]:
```suggestion
    ## 单个日志文件的大小，默认是 "100M"
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**